sink_snippets/src/g3sinks/ColorCoutSink.h

```diff
@@ -3,12 +3,28 @@
 #include <iostream>
 #include <g3log/logmessage.hpp>
 
-// NOTE: This works only on Linux/OSX
-// TODO KjellKod: For Windows terminals you can tweak this easily: https://stackoverflow.com/a/4053879/1066879
+#ifdef _WIN32
+
+#ifndef WIN32_LEAN_AND_MEAN
+    #define WIN32_LEAN_AND_MEAN 1
+#endif
+
+#include <Windows.h>
+#endif
+
 struct ColorCoutSink {
+#ifdef _WIN32
+   ColorCoutSink() : hHandle_(GetStdHandle(STD_OUTPUT_HANDLE)) {}
```

Comment:
it's annoying that WriteConsole is completely hidden in the actual output window. 
Maybe an artifact of the appveyor? 

Do you know if it's possible to see this displayed to cout? 
https://ci.appveyor.com/project/KjellKod/g3sinks/builds/45527002